src/components/Header/Header.js

```diff
@@ -79,18 +81,21 @@ const Header = ({ onSetFilter }) => {
         <NavLink to={CREATE_ACTIVITY_ROUTE} data-test="react-navlink">
           <AddIcon title="Add New Activity" />
         </NavLink>
+        <Button onClick={onStartLogout}>Logout</Button>
       </ToolsWrapper>
     </HeaderWrapper>
   );
 };
 
 Header.propTypes = {
   onSetFilter: PropTypes.func.isRequired,
+  onStartLogout: PropTypes.func.isRequired,
 };
 
-const mapDispatchToProps = {
-  onSetFilter: setFilter,
-};
+const mapDispatchToProps = dispatch => ({
```

Comment:
what is the reason of this change to function? It will work `normally` if object will be used 🤔